test/testdata/nettemplates/TwoNodesFollower100Second.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Genesis": {
+    "LastPartKeyRound": 3000,
+    "NetworkName": "tbd",
+    "Wallets": [
+      {
+        "Name": "Wallet1",
+        "Stake": 100,
+        "Online": true
+      }
+    ]
+  },
+  "Nodes": [
+    {
+      "Name": "Follower",
+      "IsRelay": true,
+      "Wallets": [],
+      "ConfigJSONOverride": "{\"EnableFollowMode\":true}"
```

Comment:
nice, I merged the code to enable this for `goal node create` networks few days ago